### ISSUE TITLE
How to use the multi gpu? 

### ISSUE THREAD

**Issue by kjh21212** [noahchalifour/rnnt-speech-recognition#6]
I have 4way 2080ti server
when i start training, i got OOM error and just gpu 0 usage 100%
so i adjusted the batchsize 64 --> 1 but the same problem occurred 

and 

"AutoGraph could not transform <bound method Popen._communicate of <subprocess.Popen object at 0x7f9665489f60>> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output."

problem occured

**Comment by noahchalifour** [comment]
I just added multi GPU support. It uses all available GPUs by default but you can also specify the number of GPUs to use by specifying the `n_gpus` argument when running. I have also moved the model to the Keras Functional API. Please check out the latest commit. 11aaae4069ea3f5f026f5b2ca81401a504dfc07f

**Comment by noahchalifour** [comment]
So, for now multi GPU won't work with the loss function. I will try and add it in the near future

**Comment by noahchalifour** [comment]
Should be resolved as of latest commit 01edf390be6b7cc52b7522d35a985c9b18419fa2.
